src/views/terms-and-conditions.html

```diff
@@ -0,0 +1,129 @@
+                </p>
+                <h4>
+                    Intellectual Property
+                </h4>
+                <p>
+                    Attendees and participants own any intellectual property they produce at the Event and no sponsors or Organisers will make a claim to Intellectual Property rights of their products or creations.
+                </p>
+                <h4>
+                    Use of Likeness
+                </h4>
+                <p>
+                    By attending Hack Cambridge you acknowledge and agree to grant the Organisers the right at Hack Cambridge to record, film, photograph, or capture your likeness in any media now available or hereafter developed and to distribute, broadcast, use, or otherwise globally to disseminate, in perpetuity, such media without any further approval from you or any payment to you. This grant to the Organisers includes, but is not limited to, the right to edit such media, the right to use the media alone or together with other information, and the right to allow others to use or disseminate the media.
+                </p>
+                <h4>
+                    Event Content
+                </h4>
+                <p>
+                    You acknowledge and agree that the Organisers, in their sole discretion, reserves the right to change any and all aspects of the Event, including but not limited to, the Event name, themes, content, programme, speakers, performers, hosts, moderators, venue, and time.
+                </p>
+                <h4>
+                    Age Requirements
+                </h4>
+                <p>
+                    No one under the age of 18 will be admitted.
+                </p>
+                <h4>
+                    Wristband Usage
+                </h4>
+                <p>
+                    Attendee's identifying wristbands must be worn at all times in Event areas. Failure to do so may result in difficulty in entering or re-entering the Premises, or refusal by security staff to allow the attendee to enter or re-enter the Premises. It may also result in expulsion from the Premises by security.
+                </p>
+                <h4>
+                    Disruptive or Unethical Conduct
+                </h4>
+                <p>
+                    You acknowledge and agree that the Organisers and other security personnel, volunteers at the Event, or staff of Cambridge Live reserve the right to remove you from the Event if the Organisers, in their sole discretion, determine that your presence or behavior create a disruption or hinder the Event or the enjoyment of the Event by other attendees or to be of an unethical or harrassing nature. All wristbands or distributed items (including swag from sponsors) must then be returned to the Organisers upon request.<br>You also agree to abide by the <br><a href="http://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank">MLH Code of Conduct</a>
```

Comment:
Full stop at end of sentence

Reply:
(relating to https://github.com/hackcambridge/hack-cambridge-website/pull/256#discussion_r141751468) Let's fix this though, as this is a regression.